lib/.eslintrc.yaml

```diff
@@ -1,4 +1,13 @@
 rules:
+  indent: [error, 2, {ArrayExpression: first,
+                  CallExpression: {arguments: first},
```

Comment:
Speaking of indentation, shouldn't these be indented a bit further to align with the third argument above it? :-)